inst/templates/methods-nab/nab-lab-methods.Rmd

```diff
@@ -0,0 +1,26 @@
+---
+title: "{{ study_name}} Lab Methods"
+output: github_document
+---
+
+<!-- This file was adapted from VISCtemplates {{pkg_ver}}. -->
+
+<!--- Refer to the protocol-specific lab study plan for the specific tiers, 
+cell types, and pseudoviruses. --->
+
+Neutralizing antibodies against HIV-1 pseudoviruses were measured as a function 
+of reduction in Tat-regulated luciferase (Luc) reporter gene expression in 
+TZM-bl cells [@Montefiori2009], [@Sarzotti-Kelsoe2014]. 
+Neutralization titer was defined as the serum dilution at which relative 
+luminescence units (RLUs) were reduced by 50% (ID~50~) relative to the RLUs in 
+virus control wells (cell + virus only) after subtraction of background RLUs 
+(wells with cells only). 
+The assay performed in TZM-bl cells measured neutralization titers against a 
+panel of <!--- autologous and ---> heterologous Env-pseudotyped viruses.
+[x] pseudoviruses were tested, consisting of [fill in virus, clade, tier], 
```

Comment:
somewhere in this file we could add a link to CATNAP - which is a reference SRAs can use to add clade and tier information. https://www.hiv.lanl.gov/components/sequence/HIV/neutralization/

Reply:
jason is helping with a new workflow to pull clade and tier from linda's nab database into pdata, so hopefully sra's don't have to pull from catnap. marie may have additional information if you're interested.

Reply:
Add a link to the lab methods template so that SRAs know that this is a resource.